src/Components/App/App.js

```diff
@@ -24,6 +25,14 @@ function App() {
     .catch(err => console.log(`There has been an error: ${err}`))
   }, [dispatch]);
 
+  useEffect(() => {
+    getFriends(1)
```

Comment:
I think it's okay to have this hard keyed for now, but we should really have this referencing `state.user.id` in our Redux slice so it's dynamic, even if we're only working with a single user right now.